Google.Api.Generator.Tests/Invoker.cs

```diff
@@ -128,5 +134,35 @@ public static WithPath TempDir()
             Directory.CreateDirectory(dirPath);
             return new WithPath(() => Directory.Delete(dirPath, recursive: true), dirPath);
         }
+
+        private static Dictionary<string, string> GetOSSpecificConfigs(string rootDir)
```

Comment:
```suggestion
        private static Dictionary<string, string> GetOsSpecificConfigs(string rootDir)
```

Reply:
I'd actually stick with OS here (like Environment.OSVersion) - two letter abbreviations are a special case as per https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers